src/main/scala/codacy/docker/api/duplication/DuplicationTool.scala

```diff
@@ -0,0 +1,24 @@
+package codacy.docker.api.duplication
+
+import codacy.docker.api.{DuplicationConfiguration, Source}
+import com.codacy.api.dtos.Language
+
+import scala.util.Try
+
+trait DuplicationTool {
+  def apply(path: Source.Directory,
+            language: Option[Language],
+            options: Map[DuplicationConfiguration.Key, DuplicationConfiguration.Value]): Try[List[DuplicationClone]]
+}
+
+object DuplicationTool {
+
+  case class Name(value: String) extends AnyVal {
+    override def toString: String = value
+  }
+
+  case class Version(value: String) extends AnyVal {
```

Comment:
Do we need this? I also think we are not using this in the metrics. Only the tools will need this. Can you check? I would like to not have stuff here that is not needed.

Reply:
Just following the pattern set by the metrics api implementation, we are not using it for anything yet. Should I remove it?